src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -16,6 +8,13 @@
 using System.Runtime.Serialization;
 using System.Runtime.Serialization.Formatters.Soap;
 using System.Text;
+using ProtoCore.DSASM;
+using ProtoCore.Exceptions;
+using ProtoCore.Lang;
```

Comment:
You can just remove this `using` as it's referencing the `TraceUtils` class. Then you won't need to prefix all usages of `TraceUtils` with `DynamoServices` namespace.

Reply:
There are other classes that are being used(in Callsite.cs) from the ProtoCore.Lang namespace and so couldn't remove this import.